bazel/test_external/BUILD

```diff
@@ -3,6 +3,9 @@ load("@emsdk//emscripten_toolchain:wasm_rules.bzl", "wasm_cc_binary")
 cc_binary(
     name = "hello-world",
     srcs = ["hello-world.cc"],
+    linkopts = [
+        "--bind",
```

Comment:
Required for embind.

https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html

Reply:
Why is this not formatted on one line like `srcs` above?  Is this the bazel style?

Reply:
```
linkopts = ["--bind"],
```
works as well. Both comply with `buildifier`.